libraries/types/vote/src/vote.cpp

```diff
@@ -33,16 +33,28 @@ bytes VrfPbftSortition::getRlpBytes() const {
 
 uint64_t VrfPbftSortition::getBinominalDistribution(uint64_t stake, double dpos_total_votes_count, double threshold,
                                                     const uint512_t& output) {
+  if (!stake) return 0;  // Stake is 0
+
   auto l = static_cast<uint512_t>(output).convert_to<boost::multiprecision::mpfr_float>();
   auto division = l / kMax512bFP;
   double ratio = division.convert_to<double>();
```

Comment:
let's improve this and make all of them `const`

Reply:
fixed